README.md

```diff
@@ -43,11 +43,11 @@ To get started with the Swift Quickstart application follow these steps:
 
 <img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
 
-2. Type in an identity and click on "Generate Access Token" from the [Testing Tools page](https://www.twilio.com/console/video/runtime/testing-tools).
+2. Generate an Access Token
```

Comment:
I noticed just the other day that we have invalid links in the source code. See here:

https://github.com/twilio/video-quickstart-ios/search?q=https%3A%2F%2Fwww.twilio.com%2Fconsole%2Fvideo%2Fruntime%2Ftesting-tools